src/domain/app/navigation/Navigation.tsx

```diff
@@ -17,27 +25,58 @@ function Navigation() {
   const navigate = useNavigate();
   const location = useLocation();
 
+  const cmsLanguageOptions = useCmsLanguageOptions();
+
+  const staticMenuItems = useStaticLinks();
+
+  const getHref = useCallback(
+    (language: LanguageCodeEnum) => {
+      const nav = cmsLanguageOptions?.find((cmsLanguageOption) => {
+        return (
+          cmsLanguageOption.locale?.toLowerCase() === language.toLowerCase()
+        );
+      });
+      const strippedPathname = stripLocaleFromUri(location.pathname);
+      // special logic for static urls
+      if (Boolean(strippedPathname)) {
```

Comment:
```suggestion
      if (strippedPathname) {
```
The explicit conversion with Boolean is not needed here, or is there something here I'm overlooking?

Reply:
forgot to remove, fixed